bcda/cclf/cclf_test.go

```diff
@@ -115,43 +133,154 @@ func (s *CCLFTestSuite) TestImportCCLF0_SplitFiles() {
 	assert.Equal(cclfFileValidator{totalRecordCount: 6, maxRecordLength: 549}, validator["CCLF8"])
 }
 
+func (s *CCLFTestSuite) TestImportCCLFDirectoryValid() {
+	assert := assert.New(s.T())
+	//Happy case, with directory containing valid BCD files.
+	_, _, _, err := ImportCCLFDirectory(filepath.Join(s.basePath, constants.CCLFDIR, "archives", "valid_bcd"))
+	assert.Nil(err)
+}
+func (s *CCLFTestSuite) TestImportCCLFDirectoryInvalid() {
+	assert := assert.New(s.T())
+	//Directory with mixed file types + at least one bad file.
+	cclfDirectory := filepath.Join(s.basePath, constants.CCLFDIR)
+	_, _, _, err := ImportCCLFDirectory(cclfDirectory)
+	assert.EqualError(err, "one or more files failed to import correctly")
+
+	//Target bad file directory
+	cclfDirectory = filepath.Join(s.basePath, constants.CCLFDIR, "archives", "invalid_bcd")
+	_, _, _, err = ImportCCLFDirectory(cclfDirectory)
+	assert.EqualError(err, "one or more files failed to import correctly")
+
+}
+
+/*
+	func (s *CCLFTestSuite) TestImportCCLFDirectoryEmpty() {
+		assert := assert.New(s.T())
+		//Zero CCLF files in directory
+		cclfDirectory := filepath.Join(s.basePath, constants.CCLFDIR, "emptydir")
+		_, _, _, err := ImportCCLFDirectory(cclfDirectory)
+		assert.Nil(err)
+	}
```

Comment:
Did this test not work? 🤔

Reply:
This one was funky; it passed locally using both go test, as well as the make unit-test, but github actions was having none of it, even after several attempts to mitigate (eg making file paths os-agnostic)